### ISSUE TITLE
BlackConnect crashes due to ActionUpdateThread.OLD_EDT being deprecated

### ISSUE THREAD

**Issue by Perlence** [lensvol/intellij-blackconnect#92]
This error started to occur randomly after upgrading to PyCharm 2024.1:

```
com.intellij.diagnostic.PluginException: `ActionUpdateThread.OLD_EDT` is deprecated and going to be removed soon. 'me.lensvol.blackconnect.actions.ReformatSelectedFragmentAction' must override `getActionUpdateThread` and chose EDT or BGT. See ActionUpdateThread javadoc. [Plugin: me.lensvol.blackconnect]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:90)
	at com.intellij.diagnostic.PluginException.reportDeprecatedUsage(PluginException.java:125)
	at com.intellij.openapi.actionSystem.ActionUpdateThreadAware.getActionUpdateThread(ActionUpdateThreadAware.java:21)
	at com.intellij.openapi.actionSystem.AnAction.getActionUpdateThread(AnAction.java:199)
```

<img width="936" alt="image" src="https://github.com/lensvol/intellij-blackconnect/assets/1044300/cf1cd855-042b-48e5-a62f-4201d84a3255">

**Comment by lensvol** [comment]
This should be fixed in release 0.6.2

We apologize for the inconvenience.

**Comment by Perlence** [comment]
Hey, don't worry about it! Thank you for maintaining the plugin!
